portal/src/test/scala/cromwell/pipeline/controller/ProjectControllerTest.scala

```diff
@@ -0,0 +1,128 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.ContentTypes.`application/json`
+import akka.http.scaladsl.model.{ HttpEntity, StatusCodes }
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectAdditionRequest, User }
+import cromwell.pipeline.service.ProjectService
+import cromwell.pipeline.utils.auth.{ AccessTokenContent, TestUserUtils }
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+import org.mockito.Mockito._
+import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.libs.json.Json
+
+import scala.concurrent.Future
+
+class ProjectControllerTest extends AsyncWordSpec with Matchers with MockitoSugar with ScalatestRouteTest {
+
+  private val projectService = mock[ProjectService]
+  private val projectController = new ProjectController(projectService)
+
+  "ProjectController" when {
+    "GET project by id" should {
+      "return project entity" in {
+        val dummyUser: User = TestUserUtils.getDummyUser(active = false)
+        val userId = dummyUser.userId
```

Comment:
This variable is excessive.
Same for `projectId`.

Reply:
fixed